flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "skip-arches": [
+        "aarch64"
```

Comment:
:-(

Reply:
i still dont know any exisiting project that use Flutter to build here. I have taken inspiration writing this template too from FluffyChat so I dont know much about Building from source.